lib/ryan.rb

```diff
@@ -18,9 +18,25 @@ class Ryan
   def_delegators :const,
     :name, :funcs, :type, :initialization_args, :func_by_name, :class?, :module?
 
+  def self.root
+    current_file_name = __FILE__
+    path_to_current_file = File.expand_path(current_file_name)
+    root_path = Pathname.new(path_to_current_file) + "../.."
+
+    return root_path
```

Comment:
More conventional and concise to simply do `Pathname.new File.expand_path('../..', __FILE__)`. You have all the right the pieces there, just organized a bit differently.